site/_config.yml

```diff
@@ -89,6 +98,11 @@ exclude:
   - /docs # ignore Basically Basic sample site
   - /example # ignore Basically Basic dev site
 
+include:
+  - content
+future: true
```

Comment:
i think there may be another place to configure this under the individual collections that would be more specific.

i think we can resolve this later. 

i'm looking in the docs to see for applying this collections, but they don't always cover everything. 

i could likely find the correct config by rendering `{{ site | jsonify }}` which I can do at home. this shows the JSON for the entire site, but it chokes on HTML pages. you just have to add it to `.json` page & layout.

Reply:
i don't think `future` can be set at the collection level, but we can instead use [published](https://jekyllrb.com/docs/front-matter/#predefined-global-variables) if we need to

Reply:
The following lines may be gratuitous.   I believe the `future: true` is necessary to enable us to render events who are in the future.
```
include:
  - content
```